src/qt/forms/sendcoinsdialog.ui

```diff
@@ -760,10 +760,30 @@
            </layout>
           </item>
           <item>
+           <widget class="QLabel" name="fallbackFeeWarningLabel">
+            <property name="toolTip">
+             <string>Using the static fallback-fee can result in sending a transaction that will take several hours or days (or never) to confirm or that you overpay the fees. Consider choosing your fee manually or wait until your have validated the complete chain.</string>
+            </property>
+            <property name="styleSheet">
+             <string notr="true">color: rgb(255, 150, 0);
```

Comment:
This affects the tooltip style as well. Additionally, it is hard to read on some styles.